pointsbot.js

```diff
@@ -3,29 +3,35 @@ const { point_types, domains, max_points, sheet_id, sheet_tab_name } = require("
 const BigNumber = require("bignumber.js");
 const { google } = require("googleapis");
 
-exports.handlePointGiving = function(auth, event, room, toStartOfTimeline, client){
+exports.handlePointGiving = function (auth, event, room, toStartOfTimeline, client) {
   if (event.getType() === "m.room.message" && toStartOfTimeline === false) {
     client.setPresence("online");
-    const message = event.getContent().body;
+    let message = event.getContent().body;
     const roomId = room.roomId;
+
+    if (message[1] === " ") {
```

Comment:
This seems like a little bit strange and unclear condition. At least write a comment what it means.

Reply:
True. It's so we can support "! dish" - requested by Kris